meta-balena-common/recipes-containers/balena/balena_git.bb

```diff
@@ -118,18 +114,12 @@ do_compile() {
 	export DOCKER_LDFLAGS="-s"
 
 	VERSION=${BALENA_VERSION} ./hack/make.sh dynbinary-balena
-
-	# Compile mobynit
-	cd .gopath/src/"${DOCKER_PKG}"/cmd/mobynit
-	go build -ldflags '-extldflags "-static ${MOBYNIT_EXTRA_LDFLAGS}"' .
-	cd -
 }
 
 do_install() {
 	mkdir -p ${D}/${bindir}
 	install -m 0755 ${S}/src/import/bundles/dynbinary-balena/balena-engine ${D}/${bindir}/balena-engine
 	install -d ${D}/boot
-	install -m 0755 ${S}/src/import/cmd/mobynit/mobynit ${D}/boot/init
 	echo ${BALENA_STORAGE} > ${D}/boot/storage-driver
```

Comment:
with mobynit gone from the recipe maybe put a comment here saying what this is for?

Reply:
hi @robertgzr sorry, I don't follow. This is removing the installation of mobynit, so where would you like to see the comment?

Reply:
this is regarding the line below where we write the `/boot/storage-driver` file

Reply:
hi @robertgzr, so, the new mobynit reads the storage driver from the hostapp config.json, however, older mobynits still need this file to boot. So a rollback to an old mobynit will still look for this file and if not found the device is bricked. Is this what you mean? I can add a brief comment explaining the above.